cd2024bfs2g1-frontend/src/main/ngx/src/app/main/pack/pack-new/add-activities/add-activities.component.ts

```diff
@@ -229,6 +229,5 @@ export class AddActivitiesComponent {
   }
 
   public goBack(): void {
-    this.router.navigate(["main/packs"]);
-  }
+    this.router.navigate(['main/packs/' + AddActivitiesComponent.packId]);  }
```

Comment:
esta variable que non sea estatica, aqui recuperadela facendo `this.packId` e os dialogos pasadeslla en `data`